src/pages/workspaces/workspace/Analyses.js

```diff
@@ -85,17 +88,28 @@ const AnalysisCard = ({
         tooltip: canWrite && 'Open without cloud compute',
         tooltipSide: 'left'
       }, [makeMenuIcon('eye'), 'Open preview']),
-      h(MenuButton, {
-        href: analysisEditLink,
-        disabled: locked || !canWrite,
-        tooltip: !canWrite && noWrite,
-        tooltipSide: 'left'
-      }, locked ? [makeMenuIcon('lock'), 'Edit (In Use)'] : [makeMenuIcon('edit'), 'Edit']),
-      h(MenuButton, {
-        href: analysisPlaygroundLink,
-        tooltip: canWrite && 'Open in playground mode',
-        tooltipSide: 'left'
-      }, [makeMenuIcon('chalkboard'), 'Playground']),
+      ...(toolLabel === tools.Jupyter.label ? [
+        h(MenuButton, {
+          href: analysisEditLink,
+          disabled: locked || !canWrite || currentRuntimeTool === tools.RStudio.label,
+          tooltip: Utils.cond([!canWrite, () => noWrite],
+            [currentRuntimeTool === tools.RStudio.label, () => 'Must have a runtime with Jupyter to edit']),
```

Comment:
To be consistent with other tooltips:

```suggestion
            [currentRuntimeTool === tools.RStudio.label, () => 'You must have a runtime with Jupyter to edit.']),
```